### PR TITLE
feat: add featured article carousel to homepage

### PR DESCRIPTION
**Pull request by abepeterkin**
Screen Shot 2021-01-01 at 11 37 12 PM

@hellatan this includes the changes you made for https://github.com/DueProcessed/dueprocessed.com/tree/hella/featured

### CODE REVIEW

**Review by hellatan — APPROVED** [review]

**Comment by abepeterkin** [comment]
Going to merge this now so we can show it at the demo tomorrow, @lilianammmatos @nameofname if either of you have feedback feel free to post it here and I'll make another PR

File: due-processed/home.php [hunk 1]
```diff
@@ -1,22 +1,17 @@
 <?php
+
+function due_processed_featured_articles_enqueue_scripts() {
```
Comment (lilianammmatos) [inline due-processed/home.php:3]: Should this function live with the `featured-articles` content part?

**Review by lilianammmatos — COMMENTED** [review]
just the one comment. also, I see that there were changes to `footenotes.php`... but github won't let me expand. what were the changes there?

**Comment by lilianammmatos** [comment]
@abepeterkin ☝️

**Pull request merged**